src/main/java/fruitshop/service/parsers/CsvParser.java

```diff
@@ -0,0 +1,7 @@
+package fruitshop.service.parsers;
+
+import java.util.List;
+
+public interface CsvParser {
+    List<List<String>> parse(String data);
```

Comment:
Can we avoid operating list of lists ? It will increase the complexity of the operations